services/rolesServices.js

```diff
@@ -0,0 +1,15 @@
+const {createRolesDB} = require("../dataAccess/roles");
+
+exports.createRoles = async (data) => {
+    const { name, scope } = data
+    const createdProduct = {
+        name,
+        scope
+    };
+    const res = await createRolesDB(createdProduct)
```

Comment:
before to create the role, check if a role with the name is already register in the database